mttl/models/expert_model.py

```diff
@@ -634,14 +635,14 @@ def training_step(self, batch, _):
         loss = super().training_step(batch, _)
         total_loss = loss.clone()
 
-        info_container = InfoContainer.get()
+        routing_gates = InfoContainer.get().routing_gates
```

Comment:
Does routing_gates always exist (getattr not required anymore)?

Reply:
yeah